src/cryptoadvance/specter/server_endpoints/wallets_api.py

```diff
@@ -67,7 +67,7 @@ def generatemnemonic():
 @login_required
 @app.csrf.exempt
 def txout_set_info():
-    res = app.specter.rpc.gettxoutsetinfo()
+    res = app.specter.rpc.gettxoutsetinfo(timeout=3600)
```

Comment:
It is more like a gimmick anyways, but if you use it to check for example whether your node is working properly it would be good to get some feedback in due time.